core/src/test/scala/unit/kafka/security/authorizer/AclAuthorizerTest.scala

```diff
@@ -988,6 +980,30 @@ class AclAuthorizerTest extends ZooKeeperTestHarness {
     }
   }
 
+  @Test
+  def testAuthorizeByResourceTypeNoAclFoundOverride(): Unit = {
+    testAuthorizeByResourceTypeNoAclFoundOverride(aclAuthorizer)
+  }
+
+  private def testAuthorizeByResourceTypeNoAclFoundOverride(authorizer: Authorizer): Unit = {
+    val props = TestUtils.createBrokerConfig(1, zkConnect)
+    props.put(AclAuthorizer.AllowEveryoneIfNoAclIsFoundProp, "true")
+
+    val cfg = KafkaConfig.fromProps(props)
+    val testAuthorizer = new AclAuthorizer
```

Comment:
can't this be `aclAuthorizer`?

Reply:
Yes. Renamed to aclAuthorizer. 
commit b6a766b